java/code/src/com/redhat/rhn/frontend/action/kickstart/tree/BaseTreeAction.java

```diff
@@ -113,6 +113,10 @@ protected ValidatorError processCommandSetters(PersistOperation operation,
                 kopts = kopts + " install=http://" + request.getLocalName() +
                     "/ks/dist/" + form.getString(LABEL);
             }
+            // disable YaST self update for SLE
+            if (!kopts.contains("self_update=")) {
+                kopts = kopts + " self_update=0 pt.options=self_update";
```

Comment:
@hustodemon : please update. Here is still the '+' sign missing. Correct would be:

```
kopts = kopts + " self_update=0 pt.options=+self_update";
```